diff --git a/node_modules/markdown-it-html5-media/dist/markdown-it-html5-media.js b/node_modules/markdown-it-html5-media/dist/markdown-it-html5-media.js
index e948eed..356b330 100644
--- a/node_modules/markdown-it-html5-media/dist/markdown-it-html5-media.js
+++ b/node_modules/markdown-it-html5-media/dist/markdown-it-html5-media.js
@@ -273,7 +273,7 @@ function renderMedia(tokens, idx, options, env, md) {
   var title = token.attrIndex('title') != -1 ? " title=\"".concat(md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1]), "\"") : '';
   var fallbackText = translate(env.language, "html5 ".concat(type, " not supported")) + '\n' + translate(env.language, 'html5 media fallback link', [url]);
   var description = token.content ? '\n' + translate(env.language, 'html5 media description', [md.utils.escapeHtml(token.content)]) : '';
-  return "<".concat(type, " src=\"").concat(url, "\"").concat(title).concat(attrs, ">\n") + "".concat(fallbackText).concat(description, "\n") + "</".concat(type, ">");
+  return "<!!".concat(type, " src=\"").concat(url, "\"").concat(title).concat(attrs, ">\n") + "".concat(fallbackText).concat(description, "\n") + "</".concat(type, ">");
 }
 /**
  * The main plugin function, exported as module.exports
diff --git a/node_modules/markdown-it-html5-media/lib/index.js b/node_modules/markdown-it-html5-media/lib/index.js
index c278b00..ba46dd6 100644
--- a/node_modules/markdown-it-html5-media/lib/index.js
+++ b/node_modules/markdown-it-html5-media/lib/index.js
@@ -288,8 +288,9 @@ function renderMedia(tokens, idx, options, env, md) {
   const url = token.attrs[token.attrIndex('src')][1];
 
   // Title is set like this: ![descriptive text](video.mp4 "title")
+  const titleContent = md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1]);
   const title = token.attrIndex('title') != -1 ?
-    ` title="${md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1])}"` :
+    ` title="${titleContent}"` :
     '';
 
   const fallbackText =
@@ -300,9 +301,9 @@ function renderMedia(tokens, idx, options, env, md) {
     '\n' + translate(env.language, 'html5 media description', [md.utils.escapeHtml(token.content)]) :
     '';
 
-  return `<${type} src="${url}"${title}${attrs}>\n` +
+  return `<figure><${type} src="${url}"${attrs}>\n` +
     `${fallbackText}${description}\n` +
-    `</${type}>`;
+    `</${type}><figcaption>${titleContent}</figcaption></figure>`;
 }
 
 
